update.sh

```diff
@@ -63,6 +63,9 @@ RUN set -xe \
 	\
 # https://github.com/docker/docker/blob/9a9fc01af8fb5d98b8eec0740716226fadb3735c/contrib/mkimage/debootstrap#L134-L151
 	&& echo 'Apt::AutoRemove::SuggestsImportant "false";' > /etc/apt/apt.conf.d/docker-autoremove-suggests
+# https://wiki.debian.org/UnattendedUpgrades
+# Enable the update/upgrade script (0=disable)
+	&& echo 'APT::Periodic::Enable "0";' > /etc/apt/apt.conf.d/docker-disable-auto-update
```

Comment:
Thank you! I've sponsored the upload with this fix in Ubuntu. This will soon be reflected in the Ubuntu Jammy build system. We could probably close this here. :)